$MirOS: ports/mailnews/pine/patches/patch-imap_src_imapd_Makefile,v 1.4 2006/03/19 18:37:25 tg Exp $

The first chunk must contain @ETC@ three times,
but /etc/nologin must not be changed (d'uh).

--- imap/src/imapd/Makefile.orig	Tue Jun 29 21:26:06 2004
+++ imap/src/imapd/Makefile	Sat Jul 18 23:31:58 2015
@@ -18,11 +18,11 @@
 # CPYRIGHT, included with this Distribution.
 
 
-ALERT=/etc/imapd.alert
+ALERT=@ETC@/imapd.alert
 USERALERT=.imapalert
 SHUTDOWN=/etc/nologin
-ANO=/etc/anonymous.newsgroups
-NNTP=/etc/imapd.nntp
+ANO=@ETC@/anonymous.newsgroups
+NNTP=@ETC@/imapd.nntp
 SHELL= /bin/sh
 
 
@@ -55,7 +55,7 @@ LDFLAGS = $(CCLIENTLIB) `cat $C/LDFLAGS`
 all:	imapd
 
 imapd: $(CCLIENTLIB) imapd.o
-	$(CC) $(CFLAGS) -o imapd imapd.o $(LDFLAGS)
+	$(CC) $(CFLAGS) -o imapd imapd.o $(LDFLAGS) $(LDSTATIC)
 
 imapd.o: $C/mail.h $C/misc.h $C/osdep.h
 
